tests/readPreference/readpreference-getTagSets-001.phpt

```diff
@@ -1,5 +1,5 @@
 --TEST--
-MongoDB\Driver\ReadPreference::getMode()
+MongoDB\Driver\ReadPreference::getTagSets()
```

Comment:
It's a stand-alone commit akin to any typo fix. The PR should make no difference.